src/components/popovers/Popover.jsx

```diff
@@ -24,6 +24,38 @@ export const getPopperPlacement = (position, alignment) => {
   return `${position}${variation}`;
 };
 
+let overflowStyle = null;
+
+const useDisableScroll = (disableScrollRef, isDisabled) => {
+  useEffect(() => {
+    if (disableScrollRef) {
+      const domNode = disableScrollRef.current;
+      if (isDisabled) {
+        overflowStyle = domNode.style.overflow;
+        domNode.style.overflow = 'hidden';
+      } else if (overflowStyle !== null) {
```

Comment:
might be able to simplify if `overflowStyle` is set outside the conditional (unless you're trying to avoid reflows by reading the property only if `isDisabled` is true)

Reply:
yeah. There's a specific reason I did this but my mind is shot right now late in the day. I can provide more context when my brain is working tomorrow.

Reply:
Yeah so now that I look at this, I can't think of a way that I would want to simplify this, as it might make it more complicated. I think the move would be to demonstrate how this works with a simplified example, and then show why the elseif isneeded here. Given that you mentioned that this is not a blocker, I'm going to move forward with merging, and then glad to chat about it tomorrow so that it makes sense (or if there's a superior way to write or document this).